README.md

```diff
@@ -77,10 +77,20 @@ SELECT min_by(x, y) FROM VALUES (1, 10), (2, 5), (3, 15), (4, 8) as tab(x, y);
 -- | 2                   |
 -- +---------------------+
 
+-- Get the kurtosis value of a column of numeric data.
+SELECT kurtosis(col) FROM VALUES (1.0), (10.0), (100.0), (10.0), (1.0) as tab(col);
+-- Results in
+- +-------------------+
+- "| kurtosis(tab.col) |"
+- +-------------------+
+- "| 4.777292927667962 |"
+- +-------------------+
+
 ```
 
 ## Done
 
 - [x] `mode(expression) -> scalar` - Returns the most frequent (mode) value from a column of data.
 - [x] `max_by(expression1, expression2) -> scalar` - Returns the value of `expression1` associated with the maximum value of `expression2`.
 - [x] `min_by(expression1, expression2) -> scalar` - Returns the value of `expression1` associated with the minimum value of `expression2`.
+- [x] `kurtosis(expression) -> scalar` - Returns the kurtosis, a measure of the `tailedness` of the distribution, for the given numeric values in `expression`.
```

Comment:
This documentation is not same as one in code, we should have consistent documentation everywhere.

Reply:
made all the mentioned changes @dharanad